packages/clerk-js/src/core/clerk.ts

```diff
@@ -1437,16 +1437,12 @@ export default class Clerk implements ClerkInterface {
     }
 
     const opts: RedirectOptions = {
-      afterSignInUrl: pickRedirectionProp('afterSignInUrl', { ctx: options, options: this.#options }, false),
-      afterSignUpUrl: pickRedirectionProp('afterSignUpUrl', { ctx: options, options: this.#options }, false),
+      afterSignInUrl: pickUrl('afterSignInUrl', [options, this.#options]),
+      afterSignUpUrl: pickUrl('afterSignUpUrl', [options, this.#options]),
```

Comment:
❓ @desiprisg is the `pickUrl()` fallbacks to `queryParams.redirect_url` or `options.redirectUrl` that the `pickRedirectionProp` was?